src/Bridge/Symfony/Bundle/Resources/config/api.xml

```diff
@@ -240,6 +240,8 @@
             <argument>%api_platform.title%</argument>
             <argument>%api_platform.description%</argument>
             <argument>%api_platform.version%</argument>
+            <argument>null</argument>
+            <argument type="expression">container.hasParameter('api_platform.swagger.versions') ? parameter('api_platform.swagger.versions') : null</argument>
```

Comment:
We don't have a hard dependency to ExpressionLanguage, it may not be installed. Just inject `%api_platform.swagger.versions%` and set it to `null` in the extension if not set (is it possible?)